### DISCUSSION TITLE
How to create volume or surface output of a variable

### DISCUSSION THREAD

**Discussion by aeroamit** [su2code/SU2#2048] [General]
Hi all,

Can someone point me for creating the volume or surface output for a new variable. For example, I want to plot a variable from a new/existing implementation of invscid or viscous flux discretization. 
Any old thread (or the files I need to make changes in)?

Thanks

**Comment by bigfooted** [comment]
You've probably found it by now, but if you look in the code for `AddVolumeOutput` and `SetVolumeOutputValue` you will find the locations where stuff is set up for volume output (like paraview).

**Comment by aeroamit** [comment]
@bigfooted 
Yep Thanks
